ci/build.sh

```diff
@@ -56,8 +51,23 @@ if [ -n "$GITHUB_TOKEN" ]; then
   git config --global user.email "travis@travis-ci.org"
   git config --global user.name "Travis CI"
   git remote add origin-repo https://${GITHUB_TOKEN}@github.com/SumoLogic/sumologic-kubernetes-collection.git > /dev/null 2>&1
+  git fetch origin-repo
+  git checkout $TRAVIS_BRANCH
+fi
+
+# Check for invalid changes to overrides yaml files
```

Comment:
should this check be at the top like it was before? The reason we put it at the top is because we didn't want to build the gems or the image if it would fail here

Reply:
If we're okay with moving the `# Set up Github` part to the top too, I'm fine with having it fail faster.  In my testing it wasn't a huge deal that it built the gems/docker image before failing.